<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="spacing">
         <number>9</number>
        </property>
        <item row="0" column="2">
         <widget class="M3DEditGUI::AxisViewWidget" name="axisXZ" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="M3DEditGUI::AxisViewWidget" name="axisXY" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="M3DEditGUI::AxisViewWidget" name="axisYZ" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="M3DEditGUI::RenderedViewWidget" name="perspective" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Properties</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Geometry Id</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="geoID">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Diffuse Color (RGBA)</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QDoubleSpinBox" name="rSpin">
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="gSpin">
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="bSpin">
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="aSpin">
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1016</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionAddGeo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBrush"/>
   <addaction name="actionCamera"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAddGeo">
   <property name="text">
    <string>AddGeo</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="actionBrush">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Brush</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>M3DEditGUI::AxisViewWidget</class>
   <extends>QWidget</extends>
   <header>axisviewwidget.h</header>
   <container>1</container>
   <slots>
    <signal>newCamPos(QVector3D,QVector3D)</signal>
    <signal>brushStop()</signal>
    <signal>brushDraw(M3DEditLevel::Box)</signal>
    <slot>cameraToggle(bool)</slot>
    <slot>geoToolToggle(bool)</slot>
    <slot>stopBrush()</slot>
    <slot>drawBrush(M3DEditLevel::Box)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>M3DEditGUI::RenderedViewWidget</class>
   <extends>QWidget</extends>
   <header>renderedviewwidget.h</header>
   <container>1</container>
   <slots>
    <slot>newCamPos(QVector3D,QVector3D)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionCamera</sender>
   <signal>toggled(bool)</signal>
   <receiver>axisXY</receiver>
   <slot>cameraToggle(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCamera</sender>
   <signal>toggled(bool)</signal>
   <receiver>axisXZ</receiver>
   <slot>cameraToggle(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCamera</sender>
   <signal>toggled(bool)</signal>
   <receiver>axisYZ</receiver>
   <slot>cameraToggle(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>475</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBrush</sender>
   <signal>toggled(bool)</signal>
   <receiver>axisXY</receiver>
   <slot>geoToolToggle(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBrush</sender>
   <signal>toggled(bool)</signal>
   <receiver>axisXZ</receiver>
   <slot>geoToolToggle(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBrush</sender>
   <signal>toggled(bool)</signal>
   <receiver>axisYZ</receiver>
   <slot>geoToolToggle(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>475</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>axisXY</sender>
   <signal>newCamPos(QVector3D,QVector3D)</signal>
   <receiver>perspective</receiver>
   <slot>newCamPos(QVector3D,QVector3D)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>546</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>axisXZ</sender>
   <signal>newCamPos(QVector3D,QVector3D)</signal>
   <receiver>perspective</receiver>
   <slot>newCamPos(QVector3D,QVector3D)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>674</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>axisYZ</sender>
   <signal>newCamPos(QVector3D,QVector3D)</signal>
   <receiver>perspective</receiver>
   <slot>newCamPos(QVector3D,QVector3D)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>axisXY</sender>
   <signal>brushDraw(M3DEditLevel::Box)</signal>
   <receiver>axisXZ</receiver>
   <slot>drawBrush(M3DEditLevel::Box)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>axisXY</sender>
   <signal>brushStop()</signal>
   <receiver>axisXZ</receiver>
   <slot>stopBrush()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>axisXZ</sender>
   <signal>brushDraw(M3DEditLevel::Box)</signal>
   <receiver>axisXY</receiver>
   <slot>drawBrush(M3DEditLevel::Box)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>axisXZ</sender>
   <signal>brushStop()</signal>
   <receiver>axisXY</receiver>
   <slot>stopBrush()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>axisXY</sender>
   <signal>brushDraw(M3DEditLevel::Box)</signal>
   <receiver>axisYZ</receiver>
   <slot>drawBrush(M3DEditLevel::Box)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>axisXY</sender>
   <signal>brushStop()</signal>
   <receiver>axisYZ</receiver>
   <slot>stopBrush()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>axisYZ</sender>
   <signal>brushDraw(M3DEditLevel::Box)</signal>
   <receiver>axisXY</receiver>
   <slot>drawBrush(M3DEditLevel::Box)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>axisYZ</sender>
   <signal>brushStop()</signal>
   <receiver>axisXY</receiver>
   <slot>stopBrush()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>axisYZ</sender>
   <signal>brushDraw(M3DEditLevel::Box)</signal>
   <receiver>axisXZ</receiver>
   <slot>drawBrush(M3DEditLevel::Box)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>axisYZ</sender>
   <signal>brushStop()</signal>
   <receiver>axisXZ</receiver>
   <slot>stopBrush()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
